using-timescaledb/reading-data.md

```diff
@@ -147,6 +147,26 @@ SELECT
   ORDER BY time
 ```
 
+### Delta [](rate)
+
+In many monitoring (and IoT) use cases, devices or sensors report metrics that typically do not
+change. In cases where the value changes, these changes are considered anomalies. When querying
+for these changes in values over time, users typically do not want to transmit all the values where
+no changes were observed since that impacts the performance of a query.
```

Comment:
the execution performance is not improved by this query, the main purpose of this optimization is minimizing data that gets transferred to the client

Reply:
Done!